Dockerfile

```
@@ -61,4 +60,5 @@ RUN pip install -r requirements.txt
 #the output is sent straight to terminal without being first buffered
 ENV PYTHONUNBUFFERED 1
 
-CMD [ "python", "application.py"]
+ENV PYTHONPATH "${PYTHONPATH}:/app"
```

We also have
```suggestion
#the output is sent straight to terminal without being first buffered
ENV PYTHONUNBUFFERED 1
ENV PYTHONPATH "${PYTHONPATH}:/app"
```
in other projects. Not sure if this is necessary though.

---

The dockerfile was tested and this was necessary 👍🏻  Most of our projects run through gunicorn from the terminal which does not have the same PYTHONPATH issue. Job-executor is the other component that runs python directly in the terminal and also has this included in the Dockerfile.

---

Oh, I meant the PYTHONUNBUFFERED env var.

---

This is a good explanation for why:
https://stackoverflow.com/questions/59812009/what-is-the-use-of-pythonunbuffered-in-docker-file

---

Thanks! So I think this is something nice to have - see the logs in real time.